discourse/janitor.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Discourse",
+  "description": "Discourse is the 100% open source discussion platform built for the next decade of the Internet.",
+  "icon": "https://janitor.technology/img/discourse.png",
+  "docker": {
+    "image": "janx/discourse"
+  },
+  "ports": {
+    "22": {
+      "proxy": "none"
+    },
+    "8088": {
+      "label": "VNC",
+      "proxy": "https"
+    },
+    "8089": {
+      "label": "Cloud9",
+      "proxy": "https"
+    },
+    "3000": {
+      "label": "Discourse",
+      "proxy": "https"
+    }
+  },
+  "scripts": {
+    "Update source code": "git fetch origin && git rebase origin/master",
+    "Send to code review": "hub pull-request",
```

Comment:
Nit: We usually place `Send to code review` at the end of the `scripts`, but it's not a hard convention yet.

Additionally, I don't like that we currently use descriptions (e.g. `Send to code review`) or explicit commands (e.g. `hub pull-request`) interchangeably, so I'd really like to change our Cloud9 plugin that reads this file so that it puts both description and command explicitely in the button (e.g. using `Description (explicit command)` format).